src/libs/ReportUtils.js

```diff
@@ -2503,6 +2503,26 @@ function isIOUOwnedByCurrentUser(report, allReportsDict = null) {
     return reportToLook.ownerAccountID === currentUserAccountID;
 }
 
+/**
+ * Should return true only for personal 1:1 report
+ *
+ * @param {Object} report (chatReport or iouReport)
+ * @returns {boolean}
+ */
+function isOneOnOneChat(report) {
+    return (
+        !isThread(report) &&
+        !isChatRoom(report) &&
+        !isExpenseRequest(report) &&
+        !isMoneyRequestReport(report) &&
+        !isPolicyExpenseChat(report) &&
+        !isTaskReport(report) && //
+        isDM(report) &&
+        !isIOUReport(report) &&
+        report.participantsList.length === 1
```

Comment:
I don't this correctly fixes the issue with showing up status icon next to channels like `#admins` - in my case there is only one admin, and the icons keeps showing up